pkgs/development/libraries/gstreamer/good/default.nix

```diff
@@ -6,6 +6,7 @@
 , libsoup, libpulseaudio, libintl
 , darwin, lame, mpg123, twolame
 , gtkSupport ? false, gtk3 ? null
+, xorg
```

Comment:
as far as I remember, `xorg` entries are also searched automatically by `callPackage`, so you can ask for `libXfixes` and `libXdamage` directly.

Reply:
Is that better though? I find it very implicit / non-obvious that this works, given that `libXfixes` is not in a top-level `all-packages` entry.

Reply:
Where is it described how that works and what it will pass?

Unless it's described somewhere so it's obvious to readers where it comes from, I'd like to use the current approach which leaves no doubt.

Reply:
`callPackage`, and especially what it actually does — let alone how, is basically not described well anywhere (I could say https://github.com/NixOS/nixpkgs/blob/1578559e65f21d34caffc885eecfda87322f4ed2/lib/customisation.nix#L87 and https://github.com/NixOS/nixpkgs/blob/master/pkgs/top-level/splice.nix#L124 and the second place does mention `xorg`, but it is not exactly easy to find without having an idea in advance)

Reply:
I take this as an argument for using the explicit approach I have for now -- unless I misunderstood.

Reply:
No, basically all of non-server Nixpkgs break if top-level `callPackage` stops passing stuff from `xorg`, so the expectation is to rely on this behviour.

Reply:
OK, I've switched to using them directly instead via `xorg`.